tests/unit_tests/test_wave_06.py

```diff
@@ -227,7 +248,16 @@ def test_swap_best_by_category_no_other_match_is_false():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    assert result == False
+    assert len(tai.inventory) ==3
+    assert len(jesse.inventory) == 3
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_c in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_f in jesse.inventory
```

Comment:
You can combine lines 252 and 253 if you want:
`assert len(tai.inventory) == 3 and len(jesse.inventory) == 3`

You can use list destructing like this too - a pythonic way of doing it:
```python
assert [item_a, item_b, item_c] == tai.inventory
assert [item_d, item_e, item_f] == jesse.inventory
```

https://medium.com/@umaramanat66/destructuring-list-in-python-like-javascript-f7d4c0968538